cmd/metal-api/internal/service/project-service.go

```diff
@@ -73,6 +76,36 @@ func (r projectResource) webService() *restful.WebService {
 		Returns(http.StatusOK, "OK", []v1.ProjectResponse{}).
 		DefaultReturns("Error", httperrors.HTTPErrorResponse{}))
 
+	ws.Route(ws.DELETE("/{id}").
+		To(admin(r.deleteProject)).
```

Comment:
LGTM, a test to prove that the permission checks work would make it perfect

Reply:
done